pkg/assets/builder.go

```diff
@@ -24,6 +24,8 @@ import (
 	"strings"
 )
 
+var YAMLSep = []byte("\n\n---\n\n")
```

Comment:
Picky comment, this is only used in one place, inside a method and seems the package level is high unless we're aiming to export and share this around in the API. 

It maybe more readable just inline, but I'm ok with this ultimately.